CIME/Tools/Makefile

```diff
@@ -300,6 +300,10 @@ ifeq ($(ULIBDEP),$(null))
       ULIBDEP += $(LIBROOT)/libatm.a
       CPPDEFS += -DATM_PRESENT
     endif
+    ifeq ($(GLC_PRESENT),TRUE)
```

Comment:
Did this need to be moved?  If not can you restore original?

Reply:
I don't know what you mean by 'moved'. I thought we needed a matching section for DGLC. Is this not the case? I think it built without this change. I will restore the original for now.